tools/stage/src/main/scala/ChipyardStage.scala

```diff
@@ -36,12 +35,12 @@ final class ChipyardChiselStage extends ChiselStage {
 }
 
 class ChipyardStage extends ChiselStage {
+  def extraPostPhases: Seq[PhaseManager.PhaseDependency] = Seq.empty
```

Comment:
Instead of defining a new Generator just for firechip, can we just a new a flag to the chipyard.Generator CLI (`--emit-legacy-sfc`) that adds the extra phase?

Users have asked about supporting SFC analysis passes in non-firesim flows. This would enable that, and save us from having to maintain 2 Generators.